src/Symfony/Component/Security/Core/Resources/translations/security.hy.xlf

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Նույնականացման սխալ։</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Նույնականացման տվյալները չեն գտնվել։</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Համակարգային սխալ՝ նույնականացման հացրման պրոցեսինգի ժամանակ։</source>
+                <target></target>
```

Comment:
You accidentally removed the source. 

```suggestion
                <source>Authentication request could not be processed due to a system problem.</source>
                <target>Համակարգային սխալ՝ նույնականացման հացրման պրոցեսինգի ժամանակ։</target>
```

Reply:
commit already processing by bots 👍

Reply:
@Nyholm is Travis CI important for translations?

Reply:
Yes, Travis does not need to be green but we need to make sure it does not fail because of our change. 

See https://github.com/symfony/symfony/pull/39809#discussion_r556336708